bndtools.m2e/src/bndtools/m2e/MavenImplicitProjectRepository.java

```diff
@@ -172,6 +173,20 @@ private void createRepo(IMavenProjectFacade projectFacade, IProgressMonitor moni
             if (logger.isErrorEnabled()) {
                 logger.error("Failed to create implicit repository for m2e project {}", getName(), e);
             }
```

Comment:
I think you should do Eclipse logging of the exception here so it shows up in the Eclipse log. I have no idea where slf4j logging goes. But people will see and can report the exception if it goes into the Eclipse log.

https://github.com/bndtools/bnd/blob/e7f814b8d335a6d1efc5241b7433852742ff956c/bndtools.builder/src/org/bndtools/builder/classpath/BndContainerInitializer.java#L60